package.json

```diff
@@ -1478,6 +1482,11 @@
                     "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsRegionFunctionNodeDownloadable)$/",
                     "group": "4@1"
                 },
+                {
+                    "command": "aws.copyLambdaUrl",
+                    "when": "view == aws.explorer && viewItem =~ /^(awsRegionFunctionNode|awsRegionFunctionNodeDownloadable)$/",
+                    "group": "2@0"
```

Comment:
This is a good idea. Though it will be a bit more work to implement this since the URL is not part of the Node instance, and instead each Node type will have their own implementation to get a URL.

I think we could do some function overloading, or something, once we start to have more copyUrl implementations.